### PR TITLE
Prometheus and Grafana work - someone should test and confirm

### PR DESCRIPTION
**Pull request by andersbjergfelt**
Go for

docker-compose down -v --rmi all --remove-orphans -> delete all images (if you have another way that's fine)

docker-compose up 

and check 

http://localhost:11501/metrics

http://localhost:9090/targets

http://localhost:3000/ -> Grafana

http://prometheus:9090 as datasource in Grafana.

### CODE REVIEW

**Comment by andersbjergfelt** [comment]
Do we need to make a new docker file for both Prometheus and Grafana? And run them on the server? I don't know if running docker-compose on the server will destroy the correct containers (and all the data in the DB)

**Comment by Trivivium** [comment]
@andersbjergfelt If I understood the docs for the SQL Server image correctly then it should retain the data as long as the container isn't destroyed completely. So restarts shouldn't affect us. 

However I'm a bit worried still, so we should maybe consider looking into some backups soon and preferably external data persistence using a volume.

File: src/docker-compose.yml [hunk 1]
```diff
@@ -17,23 +17,4 @@ services:
        SA_PASSWORD: "ULA2V9sPbG"
        ACCEPT_EULA: "Y"
 
-  prometheus:
-    image: prom/prometheus
-    restart: always
-    depends_on:
-      - webapplication
```
Comment (Trivivium) [inline src/docker-compose.yml]: I'm not too familiar with Docker Compose, but does us losing this configuration have an impact on the containers? Can they still see each other?

Comment (andersbjergfelt) [reply src/docker-compose.yml]: I just tested it. They can still see eachother

Comment (andersbjergfelt) [reply src/docker-compose.yml]: Using two different composes, that is

**Comment by andersbjergfelt** [comment]
@Trivivium true.. And we should be worried when doing deployments... :P We should look into backups asap

File: src/docker-compose.yml (hunk 1 above)
Comment (andersbjergfelt) [reply src/docker-compose.yml]: Depends_on ensures that it does in fact start before the other container. But as far I understand they still reside in the same network

**Comment by Trivivium** [comment]
> docker-compose down -v --rmi all --remove-orphans -> delete all images (if you have another way that's fine)

**Comment by andersbjergfelt** [comment]
Should only be done locally tho

**Pull request merged**